src/Input/InputEventsTask.cpp

```diff
@@ -119,7 +119,13 @@ InputEvents::eventGotoLookup([[maybe_unused]] const TCHAR *misc)
   if (protected_task_manager == NULL)
     return;
 
-  auto wp = ShowWaypointListDialog(basic.location);
+  WaypointPtr wp;
+  if (StringIsEmpty(misc)) {
+    wp = ShowWaypointListDialog(basic.location);
+  } else {
+    auto waypoints = &way_points;
+    wp = waypoints->LookupName(_T(misc));
```

Comment:
This part currently break the Windows build, can anyone help me out here?
```
src/Input/InputEventsTask.cpp: In function ‘void InputEvents::eventGotoLookup(const TCHAR*)’:
src/Input/InputEventsTask.cpp:127:32: error: ‘Lmisc’ was not declared in this scope; did you mean ‘misc’?
  127 |     wp = waypoints->LookupName(_T(misc));
      |                                ^~
make: *** [build/compile.mk:108: output/WIN64/dbg/src/Input/InputEventsTask.o] Error 1
```

Reply:
Also, does anyone know why the macos build doesn't compile?
(I thought MacOS was no longer supported by XCSoar, despite I would love to see it running).